tests/buyers/views/test_buyers.py

```diff
@@ -1969,6 +1969,14 @@ def setup_method(self, method):
                 "blah": [False, True]
             }
         ]
+        self.bad_response = {
+            "supplierName": "Tea Boy Ltd.",
+            "respondToEmailAddress": "teaboy@example.com",
+            "availability": "Tomorrow",
+            "essentialRequirementsMet": False,
```

Comment:
I believe we don't want to save the information because it's an invalid form input. Therefore they have never actually applied and so therefore the buyer would never need to access it.

Sorry what do you mean by 'term system'

In terms of the old flow, I believe you could always submit the brief response and all the data would be saved but you could submit one that had 'failed' i.e. you had no answer Yes to all the essential requirements.